.coveragerc

```diff
@@ -7,9 +7,10 @@ parallel = True
 
 [report]
 show_missing = True
-exclude_lines =
+# https://coverage.readthedocs.io/en/latest/excluding.html#advanced-exclusion
+exclude_also =
```

Comment:
Maybe we could later move this to `pyproject.toml`? Not required to pass review now.

Reply:
they seem to support it, so why not